app/scripts/background.js

```diff
@@ -788,11 +788,11 @@ browser.runtime.onInstalled.addListener(({ reason }) => {
 function setupSentryGetStateGlobal(store) {
   global.getSentryState = function () {
     const fullState = store.getState();
-    const debugState = maskObject(fullState, SENTRY_STATE);
+    const debugState = maskObject({ metamask: fullState }, SENTRY_STATE);
```

Comment:
`SENTRY_STATE` assumes that the state is flattened as it is in the UI, this change (along with the change on line 363) ensures that the state is properly structured for the `!appState?.store?.metamask?.participateInMetaMetrics` check in `beforeSend` of `setupSentry.js`